### ISSUE TITLE
Error spam

### ISSUE THREAD

**Issue by phit** [CreeperShift/WirelessCharger#8]
`Server thread/ERROR [FML]: A TileEntity type info.creepershift.wificharge.block.tile.TilePersonalCharger has throw an exception trying to write state. It will not persist. Report this to the mod author`

forge-1.12.2-14.23.0.2531-universal.jar
WirelessCharger-1.3.0.jar

**Comment by CreeperShift** [comment]
Self compiled from latest source or using Release file? I need a full log btw.

**Comment by phit** [comment]
release, the full server log is beyond 800mb I'm not uploading that, but I will see and try to catch a stacktrace, though I didn't see one last night when I first noticed this in the log

**Comment by CreeperShift** [comment]
Is the server software running Vanilla Forge? This kind of error shouldn't happen unless another mod/modloader modifies stuff beyond my control.

**Comment by phit** [comment]
yes it is, I wasn't aware of any popular forge forks, yours was also the only mod having this issue

**Comment by CreeperShift** [comment]
Sorry I was talking about something like sponge or other core mods.

**Comment by phit** [comment]
yes, I'm running SpongeForge, but no other mods in this 200+ mods modpack throw this error, which leads me to believe the issue is on your end

if you can point me to the commit 1.3.0 is based off, I'll add some debug log myself to see what is actually happening and if there's anything I can PR to fix this compatibility issue

**Comment by CreeperShift** [comment]
Can you verify if the personal charger actually loses its data upon
restarting? This hasn't happened on any non sponge modpacks so I'm.
Assuming sponge is handling something differently. Check if the data is
actually lost or its just throwing a false error

**Comment by ouroborus** [comment]
[22:57:51] [Server thread/ERROR] [FML]: A TileEntity type info.creepershift.wificharge.block.tile.TilePersonalCharger has throw an exception trying to write state. It will not persist. Report this to the mod author
    java.lang.NullPointerException: null

I get this message repeated in the log every 15 seconds. It's repeated for every charger setup in the world.

All The Mods 3 5.5
WirelessCharger 1.3.0
